src/gui/mzroll/mzroll.pro

```diff
@@ -79,6 +79,7 @@ CONFIG(release, debug|release) {
 INCLUDEPATH +=  /usr/include/x86_64-linux-gnu/qt5/QtXml/ /usr/include/x86_64-linux-gnu/qt5/QtSql
 
 INCLUDEPATH +=  $$top_srcdir/src/core/libmaven  \
+                $$top_srcdir/src/common         \
```

Comment:
We had not added this include path earlier because all headers from the "common" namespace were included using `#include common/headername.h`. It was more indicative of the nature of library. But since we are now adding this new path, let's make sure that the include patterns are not a mix of both (i.e. - check if some places have "`common/headername.h`" and some just "`headername.h`").